### ISSUE TITLE
Altmetric bookmarklet doesn't pick up on the DOI

### ISSUE THREAD

**Issue by cranmer** [openjournals/brief-ideas#76]
I'm not sure if that means other similar services are also missing out on the DOI, but thought I'd give you a heads up.

**Comment by arfon** [comment]
Do we know if Impact Story picks up Zenodo DOIs in general? / cc @hpiwowar

**Comment by cranmer** [comment]
Yes https://impactstory.org/KyleCranmer/products/software-products
